functions_framework/pubspec.yaml

```diff
@@ -10,18 +10,18 @@ environment:
 dependencies:
```

Comment:
revert this change.

Reply:
@kevmoo Can you explain why?

Reply:
The scope of this PR should just be examples, right? Not changing the published packages – at least that's what I thought